relayer/crates/starknet-cli/examples/bootstrap_ibc.rs

```diff
@@ -0,0 +1,302 @@
+use hermes_starknet_chain_context::contexts::chain::StarknetChain;
+use hermes_starknet_cli::contexts::app::StarknetApp;
+use hermes_starknet_relayer::contexts::cosmos_to_starknet_relay::CosmosToStarknetRelay;
+use hermes_starknet_relayer::contexts::starknet_to_cosmos_relay::StarknetToCosmosRelay;
+use hex::FromHex;
+use ibc::core::connection::types::version::Version as IbcConnectionVersion;
+use ibc::core::host::types::identifiers::PortId;
+use starknet::core::types::Felt;
+use starknet::macros::short_string;
+use tracing::info;
+
+pub const COSMOS_HD_PATH: &str = "m/44'/118'/0'/0/0";
+
+// https://github.com/osmosis-labs/testnets
+pub const OSMOSIS_TESTNET_URL: &str = "rpc.testnet.osmosis.zone:443";
+pub const OSMOSIS_TESTNET_CHAIN_ID: &str = "osmo-test-5";
+pub const OSMOSIS_TOKEN: &str = "uosmo";
+
+// https://docs.starknet.io/tools/fullnodes-rpc-providers/#open_endpoints
+pub const STARKNET_TESTNET_URL: &str = "https://starknet-sepolia.public.blastapi.io/rpc/v0_7";
+pub const STARKNET_TESTNET_CHAIN_ID: Felt = short_string!("SN_SEPOLIA");
+// https://docs.starknet.io/chain-info
+pub const STARKNET_STRK: Felt =
+    Felt::from_hex_unchecked("0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d");
+pub const STARKNET_ETH: Felt =
+    Felt::from_hex_unchecked("0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7");
+
+pub const WASM_CODE_HASH_HEX: &str =
+    "6be4d4cbb85ea2d7e0b17b7053e613af11e041617bdb163107dfd29f706318ef";
+
+fn main() -> Result<(), Error> {
+    let config_file = std::env::args().nth(1).expect("config file path");
+
+    let runtime = init_test_runtime();
+
+    runtime.runtime.clone().block_on(async move {
+        let (starknet_chain, cosmos_chain) = {
+            info!("local environment detected. using real testnets.");
+
+            let starknet_app = StarknetApp {
+                config_path: config_file.into(),
+                runtime: runtime.clone(),
+            };
+
+            let starknet_builder = starknet_app.load_builder().await?;
+
+            let starknet_chain: StarknetChain = starknet_builder
+                .build_chain(&STARKNET_TESTNET_CHAIN_ID.to_string().parse()?)
+                .await
+                .map_err(|e| eyre::eyre!("failed to build starknet chain: {:?}", e))?;
+
+            info!(
+                "starknet_chain_status: {:?}",
+                starknet_chain
+                    .query_chain_status()
+                    .await
+                    .map_err(|e| eyre::eyre!("failed to build starknet chain: {:?}", e))?
+            );
+
+            let cosmos_chain: CosmosChain = starknet_builder
+                .cosmos_builder
+                .build_chain(&OSMOSIS_TESTNET_CHAIN_ID.parse()?)
+                .await?;
+
+            info!(
+                "cosmos_chain_status: {:?}",
+                cosmos_chain.query_chain_status().await?
+            );
```

Comment:
@ljoss17 do you know why these `info` are not printed? I used at the beginning.

```rs
let runtime = init_test_runtime();
```

Note, this is not `cfg(test)`.

Reply:
This is odd, do you see `INFO initialized Hermes test runtime`?